Tests/test_SeqIO_Gck.py

```diff
@@ -380,6 +380,17 @@ def test_conflicting_lengths(self):
         h.close()
 
 
+class TestGckWithImproperHeader(unittest.TestCase):
+
+    def test_read(self):
+        """Read a file with an incomplete header."""
+        with open('Gck/improperHeader.gck', 'rb') as f:
+            h = BytesIO(f.read())
+            with self.assertRaisesRegexp(ValueError, "Improper header, cannot read 24 bytes from handle"):
+                SeqIO.read(h, 'gck')
+            h.close()
+
+
```

Comment:
There is no reason to use ``BytesIO`` if you have a file on disk, so initially I was going to suggest removing ``BytesIO`` and passing the handle directly to ``SeqIO.read()``.

However, given ``Tests/Gck/improperHeader.gck`` is just a tiny irrelevant file, why not remove it and use ``BytesIO`` as follows:

```python
    def test_read(self):
        """Read a file with an incomplete header."""
        handle = BytesIO(b"tiny")
        with self.assertRaisesRegexp(ValueError, "Improper header, cannot read 24 bytes from handle"):
            SeqIO.read(handle, 'gck')
        handle.close()
```